block-explorer/backend/Cargo.toml

```diff
@@ -0,0 +1,63 @@
+[package]
+name = "block-explorer-backend"
+authors = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+rust-version = { workspace = true }
+version = { workspace = true }
+resolver = "2"
+publish = false
+
+[[bin]]
+name = "block-explorer-backend"
+path = "src/bin/main.rs"
+
+[dependencies]
+anyhow = { version = "1" }
+async-trait = { workspace = true }
+axum = { version = "0.6" }
+clap = { workspace = true, features = ["derive", "env"] }
+hex = "0.4"
+indoc = "2"
+jsonrpsee = { version = "0.20", features = ["client", "client-ws-transport-native-tls"] }
+prometheus = "0.13"
+serde = { version = "1", features = ["derive"] }
+serde_json = { version = "1" }
+serde_urlencoded = "0.7"
+serde_with = "3"
+sov-celestia-adapter = { path = "../../adapters/celestia", features = ["native"], version = "0.2" }
```

Comment:
It would be good to make the code independent on a particular DA. Can we make it genric over `DaSpec`?

Reply:
Done!